modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/systemview/SystemViewCommand.java

```diff
@@ -0,0 +1,223 @@
+
+    /**
+     * Argument for the system view content obtainig task.
+     *
+     * @see VisorSystemViewTask
+     */
+    private VisorSystemViewTaskArg taskArg;
+
+    /** ID of the node to get the system view content from. */
+    private UUID nodeId;
+
+    /** {@inheritDoc} */
+    @Override public Object execute(GridClientConfiguration clientCfg, Logger log) throws Exception {
+        try {
+            VisorSystemViewTaskResult res;
+
+            try (GridClient client = Command.startClient(clientCfg)) {
+                res = executeTaskByNameOnNode(
+                    client,
+                    VisorSystemViewTask.class.getName(),
+                    taskArg,
+                    nodeId,
+                    clientCfg
+                );
+            }
+
+            if (res != null)
+                printSystemViewContent(res, log);
+            else
+                log.info("No system view with specified name was found [name=" + taskArg.systemViewName() + "]");
+
+            return res;
+        }
+        catch (Throwable e) {
+            log.severe("Failed to perform operation.");
+            log.severe(CommandLogger.errorMessage(e));
+
+            throw e;
+        }
+    }
+
+    /**
+     * Prints system view content obtained via {@link VisorSystemViewTask} execution.
+     *
+     * @param taskRes Result of {@link VisorSystemViewTask} execution.
+     * @param log Logger.
+     */
+    private void printSystemViewContent(VisorSystemViewTaskResult taskRes, Logger log) {
+        List<String> colTitles = taskRes.systemViewAttributes();
+
+        List<? extends List<?>> sysViewRows = taskRes.systemViewContent();
+
+        List<Integer> colLenghts = colTitles.stream().map(String::length).collect(Collectors.toList());
+
+        List<List<String>> rows = new ArrayList<>(sysViewRows.size());
+
+        sysViewRows.forEach(sysViewRow -> {
+            ListIterator<Integer> colLenIter = colLenghts.listIterator();
+
+            rows.add(sysViewRow.stream().map(attr -> {
+                String colVal = String.valueOf(attr);
+
+                colLenIter.set(Math.max(colLenIter.next(), colVal.length()));
+
+                return colVal;
+            }).collect(Collectors.toList()));
+        });
+
+        printRow(colTitles, nCopies(colTitles.size(), STRING), colLenghts, log);
+
+        rows.forEach(row -> printRow(row, taskRes.systemViewAttributeTypes(), colLenghts, log));
+    }
+
+    /**
+     * Prints row content with respect to column types and lengths.
+     *
+     * @param row Row which content should be printed.
+     * @param types Column types in sequential order for decent row formatting.
+     * @param lengths Column lengths in sequential order for decent row formatting.
+     * @param log Logger.
+     */
+    private void printRow(
+        Collection<String> row,
+        Collection<SimpleAttributeType> types,
+        Collection<Integer> lengths,
+        Logger log
+    ) {
```

Comment:
we can add an assertion here to ensure code reader that the sizes of collections are equal